src/main/java/net/sf/jabref/logic/importer/fetcher/IsbnFetcher.java

```diff
@@ -43,6 +49,9 @@
 
     private static final String URL_PATTERN = "http://www.ebook.de/de/tools/isbn2bibtex?";
 
+    private final ProtectTermsFormatter protectTermsFormatter = new ProtectTermsFormatter(new ProtectedTermsLoader(new ProtectedTermsPreferences(ProtectedTermsLoader.getInternalLists(), Collections.emptyList(), Collections.emptyList(), Collections.emptyList())));
```

Comment:
I wasn't sure about that. They caused problems during the tests. After I found a way that the tests worked again I thought I add them. But I don't like them in the fetchers, too.